packages/wp-dashboard/src/components/editorSettings/archive/index.js

```diff
@@ -217,11 +221,21 @@ export default function ArchiveSettings({
                 ),
               }}
             >
-              {sprintf(
-                /* translators: %s: archive url. */
-                __('Visit archive page at <a>%s</a>.', 'web-stories'),
-                archiveURL
-              )}
+              {archivePageId
```

Comment:
```suggestion
              {archivePageId && archiveURL !== archiveURLRaw
```

In case the custom page archive URL is the same as the default archive URL.